taqueria-vscode-extension/src/lib/helpers.ts

```diff
@@ -847,10 +847,11 @@ export class VsCodeHelper {
 			if (!shouldOutput(outputLevel, this.logHelper.logLevel)) {
 				return;
 			}
-			let text: string;
+			let text: string | undefined = undefined;
 			if (instanceOfHasToString(err)) {
 				text = err.toString();
-			} else {
+			}
+			if (text === undefined || text === '[object Object]') {
```

Comment:
Unsure about comparing against `[object Object]`. 🤔 Is there another way?

Reply:
There can be. I'm now aware of. I could exclude all things that their constructor is `Object`, but was not sure if that's a better idea.

Reply:
Looking for a string "[object Object]" seems like code smell. This string is displayed to us due to some formatting issue. Not sure what "I could exclude all things that their constructor is Object, but was not sure if that's a better idea." means but it sounds better.